automation/README.md

```diff
@@ -0,0 +1,86 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
```

Comment:
There is also `stdci.yaml` in the root of the repository.
This file contains the list of scripts that Jenkins should run (the scripts will run in parallel).
Sorry for not mentioning it in my previous comments.

Reply:
I will be easy to understand which versions of openshift/k8s the CI runs from that file.

Reply:
I addressed it